settings.gradle

```diff
@@ -2,16 +2,32 @@ include ':app', ':sageresearch-mpower', ':researchstack-mpower'
 
 // uncomment this for a "composite build" which substitutes SageResearch artifacts
 // for the source code in the indicated directory
-//includeBuild ('../SageResearch-Android')
+includeBuild ('../SageResearch-Android') {
```

Comment:
Comment these out, but leave them in place once `SageResearch-Android` and `BridgeSDK-Android` have been published.